modules/local/summarise_for_dram_input.nf

```diff
@@ -3,18 +3,15 @@ process SUMMARISE_FOR_DRAM_INPUT {
     label 'process_single'
 
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'oras://community.wave.seqera.io/library/python:3.13.1--9856f872fdeac74e':
-        'community.wave.seqera.io/library/python:3.13.1--d00663700fcc8bcf' }"
+        'oras://community.wave.seqera.io/library/pandas:2.2.3--e136a7b7218cc69c':
+        'community.wave.seqera.io/library/pandas:2.2.3--9b034ee33172d809' }"
 
     input:
-    tuple val(meta), path(ko_summaries)
-    tuple val(meta), path(ko_per_contigs)
-    tuple val(meta), path(interpro_summaries)
-    tuple val(meta), path(dbcan_overviews)
+    tuple val(meta), path(fasta), path(ko_per_contigs_tsv), path(interproscan_tsv), path(dbcan_overview)
 
```

Comment:
Double check the subwf because the fasta file input is missing.

Reply:
yup

Reply:
Fixed https://github.com/EBI-Metagenomics/assembly-analysis-pipeline/pull/13/commits/5d0476936f63477d90063e823438be42877a1f75